src/data.js

```diff
@@ -1,6 +1,94 @@
 import BigNumber from "bignumber.js";
 import * as Constants from "./constants";
 
+export function getContractFunctionIndex(contract_type, function_type) {
+    switch (contract_type) {
+        case 'TOKEN':
```

Comment:
token separates to `TOKEN` and `SPLITTABLE_TOKEN`. Put into "contract_type.js" with contract type value.

```
export const TOKEN = 'TokenContract'
export const SPLITTABLE_TOKEN = 'TokenContractWithSplit'
export const SYSTEM = 'SystemContract'
export const LOCK = 'LockContract'
export const PAYMENT_CHANNEL = 'PaymentChannelContract'
export const NFT = 'NonFungibleContract'
```